### ISSUE TITLE
No issue

### ISSUE THREAD

**Issue by R3tr0BoiDX** [phlipsi/lightoff#1]
But you used my tiles, lol

**Comment by phlipsi** [comment]
Yes, thank you for providing these tiles! Do you feel properly attributed in the README file?

**Comment by R3tr0BoiDX** [comment]
Yus, yus, you're fine. I just stumbled across by accident. Doesn't happen to often, heh. But you can link my github page on my username if you want to, but no need :) gonna try out your game one day
